src/main/resources/application.yaml

```diff
@@ -96,7 +96,7 @@ idam:
     microservice: rd_judicial_api
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
   s2s-authorised:
-    services: ${JRD_S2S_AUTHORISED_SERVICES:rd_judicial_api,am_org_role_mapping_service,iac}
+    services: ${JRD_S2S_AUTHORISED_SERVICES:rd_judicial_api,am_org_role_mapping_service,iac,xui_webapp}
```

Comment:
We maintain the whitelisting list in flux and that takes precedence. So please move this to flux.

Reply:
Good catch, I will add in flux as well but I think I can't remove this as we need for Preview?

Reply:
for preview it takes aat flux values

Reply:
fine then. will remove. thanks

Reply:
created flux changes, removing this will be addressed in https://tools.hmcts.net/jira/browse/RDCC-3347